pkgs/initiator/initiator.go

```diff
@@ -751,3 +949,22 @@ func (c *Initiator) GetThreshold(ids []uint64) (int, error) {
 	threshold := len(ids) - ((len(ids) - 1) / 3)
 	return threshold, nil
 }
+
+func (c *Initiator) JoinSets(oldOperators []*wire.Operator, newOperators []*wire.Operator) []*wire.Operator {
```

Comment:
no reason that this func will be a member of `Initiator`. it can be stand alone function and more testable like that.